app/mailers/patient_mailer.rb

```diff
@@ -38,9 +38,15 @@ def enrollment_sms_text_based(patient)
 
     lang = patient.select_language(:sms)
     custom_msg = patient&.jurisdiction&.custom_messages_for_hierarchy&.dig('assessments', 'twilio', 'sms', 'prompt', 'intro', lang&.to_s)
-    contents = custom_msg.present? ?
-      "#{patient&.initials_age('-')},\n#{custom_msg}" :
-      I18n.t('assessments.twilio.sms.prompt.intro', locale: lang, name: patient&.initials_age('-'))
+    contents = if custom_msg.present?
+                 "#{patient&.initials_age('-')},\n#{custom_msg}" +
+                   (custom_msg.exclude?(ADMIN_OPTIONS['privacy_policy_url']) && ADMIN_OPTIONS['privacy_policy_url'].present? ?
```

Comment:
This check doesn't seem to be working (I'm getting double privacy statement in my message). But also if you agree with my other comment, this should be removed anyway.
```suggestion
                   ADMIN_OPTIONS['privacy_policy_url'].present? ?
```

Reply:
it might be because the `privacy_policy_url` currently defined might be different than the one in the custom message (in which case, it will show up twice but with different URLs, but this theoretically shouldn't happen anyways) @hackrm and I both ran into this while testing

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1247/commits/8fac5befa1faba9061b8ef057506327d991f57f2